exercises/practice/yacht/test_yacht.c

```diff
@@ -0,0 +1,250 @@
+#include "yacht.h"
+
+#include "test-framework/unity.h"
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static void test_yacht(void)
+{
+   const dice_t dice = { 5, 5, 5, 5, 5 };
+   TEST_ASSERT(score(dice, YACHT) == 50);
+}
+
+static void test_not_yacht(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 1, 3, 3, 2, 5 };
+   TEST_ASSERT(score(dice, YACHT) == 0);
+}
+
+static void test_ones(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 1, 1, 1, 3, 5 };
+   TEST_ASSERT(score(dice, ONES) == 3);
+}
+
+static void test_ones_out_of_order(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 3, 1, 1, 5, 1 };
+   TEST_ASSERT(score(dice, ONES) == 3);
+}
+
+static void test_no_ones(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 4, 3, 6, 5, 5 };
+   TEST_ASSERT(score(dice, ONES) == 0);
+}
+
+static void test_twos(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 2, 3, 4, 5, 6 };
+   TEST_ASSERT(score(dice, TWOS) == 2);
+}
+
+static void test_fours(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 1, 4, 1, 4, 1 };
+   TEST_ASSERT(score(dice, FOURS) == 8);
+}
+
+static void test_yacht_counted_as_threes(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 3, 3, 3, 3, 3 };
+   TEST_ASSERT(score(dice, THREES) == 15);
+}
+
+static void test_yacht_of_3s_counted_as_fives(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 3, 3, 3, 3, 3 };
+   TEST_ASSERT(score(dice, FIVES) == 0);
+}
+
+static void test_fives(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 1, 5, 3, 5, 3 };
+   TEST_ASSERT(score(dice, FIVES) == 10);
+}
+
+static void test_sixes(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 2, 3, 4, 5, 6 };
+   TEST_ASSERT(score(dice, SIXES) == 6);
+}
+
+static void test_full_house_two_small_three_big(void)
+{
+   TEST_IGNORE();
+   const dice_t dice = { 2, 2, 4, 4, 4 };
+   TEST_ASSERT(score(dice, FULL_HOUSE) == 16);
```

Comment:
For the test we tend to use the type specific test macros from Unity rather than TEST_ASSERT(). Can you check if there are any suitable for this exercise?